### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by andykillen**
Added install instructions

Added instructions on how to adapt this for Sapper use, forcing client side rendering only.

### CODE REVIEW

**Comment by antony** [comment]
Thanks for this.

I've requested a few changes :)

File: README.md [hunk 1]
```diff
@@ -11,6 +11,9 @@ Google and Facebook Auth
 SSR Ready
 Svelte v3 + v2
 
+## installation
+````npm i @beyonk/svelte-social-auth````
```
Comment (antony) [inline README.md:15]: Svelte applications are bundled, so dependencies should **always** be installed as devDependencies - you should add -D to this command.

File: README.md [hunk 2]
```diff
@@ -33,6 +36,27 @@ Svelte v3 + v2
 </script>
 ```
 
+## Adaption for Sapper
```
Comment (antony) [inline README.md:39]: I wrote this module for Sapper. I use it in Sapper without this adaptation (and I wrote that section of the Sapper docs!) - are you sure it's necessary?

If it is perhaps I can help you debug. It shouldn't be.

**Review by antony — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (andykillen) [reply README.md:15]: @antony its a direct copy of the install instructions on npm.  and perhaps also why its failing SSR build on sapper. 

again a good reason to have install instructions.

File: README.md (hunk 2 above)
Comment (andykillen) [reply README.md:39]: I'll re-do the install with devDependencies, and if its still a problem, I'll make a video of the problem.  In its current state, its deffo not working for me. 

I also did see one other change I forgot to add to this pull request, the alt text of the facebook icon is "google", will also do that tomorrow.

File: README.md (hunk 1 above)
Comment (andykillen) [reply README.md:15]: @antony it was the --save-dev switch that was needed to make this work as an SSR component out of the box

File: README.md (hunk 2 above)
Comment (andykillen) [reply README.md:39]: @antony as described above. the --save-dev made all the difference. I'm gonna adapt the pull request to remove this, adapt the install instructions and change "google" to "facebook" in the <FacebookAuth> component.

**Pull request merged**